utils/time.go

```diff
@@ -3,13 +3,12 @@ package utils
 import (
 	"time"
 
+	"github.com/golang/protobuf/ptypes"
 	"github.com/golang/protobuf/ptypes/timestamp"
 )
 
 // ToTimestamp converts time.Time to timestamp.TimeStamp.
 func ToTimestamp(time time.Time) *timestamp.Timestamp {
-	return &timestamp.Timestamp{
-		Seconds: int64(time.Second()),
-		Nanos:   int32(time.Nanosecond()),
-	}
+	ts, _ := ptypes.TimestampProto(time)
```

Comment:
there wont be any error from this func ideally. But lets `panic` here instead of passing the error back just to be sure.